docs/extend/extend-apiml/api-mediation-personal-access-token.md

```diff
@@ -0,0 +1,151 @@
+Such criteria define the level of access control and can restrict the access in advance. This can be done by using
+**rules**, and the access restriction can be applied by either user ID or service scopes.
+
+The administrator can invalidate all the tokens bound to a specific user by calling the following REST API endpoint via Gateway:
+
+`DELETE /auth/access-token/revoke/tokens/users`
+The full path of the `/auth/access-token/revoke/tokens/users` endpoint appear as `https://{gatewayUrl}:{gatewayPort}/gateway/api/v1/auth/access-token/revoke/tokens/users`.
+
+The request requires the body in the following format:
+
+```json
+{
+   "userId": "<userId>",
+   "timestamp": "<timestamp>"
+}
+```
+
+The `userId` is the user the revocation should be applied to, while the timestamp represents the current date of revocation, in milliseconds. 
+
+When invalidation is successful, the response to the request is an empty body.
+
+## Invalidate all the tokens for a service
+
+As we mentioned in the above section, it is possible to revoke tokens also by using rules based on service scopes.
+The administrator can invalidate all the tokens bound to a specific service by calling the following REST API endpoint via Gateway:
+
+`DELETE /auth/access-token/revoke/tokens/scope`
+The full path of the `/auth/access-token/revoke/tokens/scope` endpoint appear as `https://{gatewayUrl}:{gatewayPort}/gateway/api/v1/auth/access-token/revoke/tokens/scope`.
+
+The request requires the body in the following format:
+
+```json
+{
+   "serviceId": "<serviceId>",
+   "timestamp": "<timestamp>"
+}
+```
+
+The `serviceId` represents the service the revocation should be applied to (e.g. APPL IDs), while the timestamp represents the current date of revocation, in milliseconds.
+
+When invalidation is successful, the response to the request is an empty body.
+
+## Retrieve all the tokens and rules
+
+The user can retrieve all the invalidated tokens and rules which were stored in the cache by calling the following REST API endpoint via Gateway:
+
+`GET /cache-list`
+The full path of the `/cache-list` endpoint appear as `https://{gatewayUrl}:{gatewayPort}/gateway/api/v1/cache-list`.
+
+When retrieval is successful, the response to the request is a body containing all the key/value entries presents in the cache.
+
+## Evict the non-relevant tokens and rules
+
+The Security Administrator can evict the non-relevant invalidated tokens and rules from the cache by calling the following REST API endpoint via Gateway:
+
+`DELETE /auth/access-token/evict`
+The full path of the `/auth/access-token/evict` endpoint appear as `https://{gatewayUrl}:{gatewayPort}/gateway/api/v1/auth/access-token/evict`.
+
+The `/auth/access-token/evict` endpoint evicts the all the invalidated tokens which were expired and all the rules related to the expired tokens.
+
+The main purpose of the eviction API is to make sure that the size of the cache does not grow unbounded.
```

Comment:
And this isn't due to the actual size on disk but for the verification that the tokens are valid all the rules needs to be consulted.